rxui/src/test/java/com/artemzin/rxui/RxUiTest.java

```diff
@@ -9,13 +9,14 @@
 
 import java.util.concurrent.atomic.AtomicBoolean;
 
-import rx.Observable;
-import rx.Subscription;
-import rx.functions.Action1;
-import rx.functions.Func1;
+import io.reactivex.Observable;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.functions.Consumer;
+import io.reactivex.functions.Function;
 
 import static java.util.Arrays.asList;
-import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
They won't work but the Java6Assertions will `import static org.assertj.core.api.Java6Assertions.assertThat` (then you might be able to use them again if you had weird errors before)